inst/extdata/config.yaml

```diff
@@ -0,0 +1,13 @@
+default:
+  base_path: "https://trace.virtual.uniandes.edu.co/colopendata"
```

Comment:
> I would be very interested to have some documentation (maybe in a vignette) indicating how long we expect these data to be available + why we have that confidence.
> 
> If people build on this package, the URL here is a major breaking point (single point of failure). If something happens with this URL, all the data this package builds on becomes unavailable. As such, I consider we have a responsibility to provide some more documentation on this.
> 
> What do you think?

The current server is temporary and will be used only in this early stage for initial user testing and package development. In the future, we are looking forward to store the data in a server mantained by the Information and Technology Services Direction (DSIT in spanish) from Universidad de los Andes. This will provide more stability and durability. We will be working on this during the next couple months.